logging/slog_logger.go

```diff
@@ -0,0 +1,71 @@
+package logging
+
+import (
+	"fmt"
+	"log/slog"
+	"os"
+)
+
+type SLogger struct {
+	logger *slog.Logger
+}
+
+var _ Logger = (*SLogger)(nil)
+
+func NewSlogLogger(env LogLevel) Logger {
+	if env == Production {
+		logger := slog.New(slog.NewJSONHandler(os.Stdout, nil))
+		return &SLogger{
+			logger: logger,
+		}
+	} else if env == Development {
+		logger := slog.New(slog.NewTextHandler(os.Stdout, nil))
```

Comment:
I guess dev should also be json handler? since we see logs on grafana or anywhere

Reply:
Is TextHandler the same as logfmt?

Reply:
added

Reply:
@dmanc texthandler gives in this format 
`time=2022-11-08T15:28:26.000-05:00 level=INFO msg=hello count=3`

Reply:
I know we don't have structured logging in grafana yet (or we do? ) but I think structured logging needs json?

Reply:
Yeah texthandler is the same as logfmt it seems. Loki supports either:
- https://grafana.com/docs/loki/latest/send-data/promtail/stages/logfmt/
- https://grafana.com/docs/loki/latest/send-data/promtail/stages/json/

Reply:
that's cool. we can do either but should follow a standard everywhere.